addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -87,7 +87,7 @@ html[data-edit_translations="1"] {
         display: inline-flex;
         margin: 1% 0;
         padding-right: 0.3rem;
-        width: 50%;
+        width: 100%;
```

Comment:
As discussed, we'll keep the current "3 colors" palette preview.
Could you update the screenshot in the PR?
Thanks